Import noble/hashes not expose sha2

Hi!

I got this error with last version.

```bash
Error: Cannot find module './node_modules/.pnpm/@polkadot-labs+hdkd-helpers@0.0.23/node_modules/@noble/hashes/sha2' imported from ./node_modules/.pnpm/@polkadot-labs+hdkd-helpers@0.0.23/node_modules/@polkadot-labs/hdkd-helpers/dist/index.js
Did you mean to import "@noble/hashes/sha2.js"?
```

--- a/packages/hdkd-helpers/src/bip39.ts
+++ b/packages/hdkd-helpers/src/bip39.ts
@@ -2,8 +2,8 @@
 //
 // Adapted from https://github.com/bitcoinjs/bip39/blob/a7ecbfe2e60d0214ce17163d610cad9f7b23140c/ts_src/index.ts
 
-import { sha256 } from "@noble/hashes/sha2"
-import { randomBytes } from "@noble/hashes/utils"
+import { sha256 } from "@noble/hashes/sha2.js"
+import { randomBytes } from "@noble/hashes/utils.js"
 import { BIP39_EN_WORDLIST } from "./bip39EnWordlist"
 
 const INVALID_MNEMONIC = "Invalid mnemonic"

--- a/packages/hdkd-helpers/src/createChainCode.test.ts
+++ b/packages/hdkd-helpers/src/createChainCode.test.ts
@@ -1,4 +1,4 @@
-import { bytesToHex } from "@noble/hashes/utils"
+import { bytesToHex } from "@noble/hashes/utils.js"
 import { describe, expect, test } from "vitest"
 import { createChainCode } from "./createChainCode"
 

--- a/packages/hdkd-helpers/src/createDerive.test.ts
+++ b/packages/hdkd-helpers/src/createDerive.test.ts
@@ -1,4 +1,4 @@
-import { bytesToHex } from "@noble/hashes/utils"
+import { bytesToHex } from "@noble/hashes/utils.js"
 import { expect, test } from "vitest"
 import { type Curve, ecdsa, ed25519, type Hex } from "."
 import { DEV_PHRASE } from "./constants"

--- a/packages/hdkd-helpers/src/substrateBip39.test.ts
+++ b/packages/hdkd-helpers/src/substrateBip39.test.ts
@@ -1,4 +1,4 @@
-import { bytesToHex } from "@noble/hashes/utils"
+import { bytesToHex } from "@noble/hashes/utils.js"
 import { expect, test } from "vitest"
 import { DEV_MINI_SECRET, DEV_PHRASE } from "./constants"
 import {

--- a/packages/hdkd-helpers/src/substrateBip39.ts
+++ b/packages/hdkd-helpers/src/substrateBip39.ts
@@ -1,5 +1,5 @@
-import { pbkdf2, pbkdf2Async } from "@noble/hashes/pbkdf2"
-import { sha512 } from "@noble/hashes/sha2"
+import { pbkdf2, pbkdf2Async } from "@noble/hashes/pbkdf2.js"
+import { sha512 } from "@noble/hashes/sha2.js"
 import { mnemonicToEntropy } from "./bip39"
 import { BIP39_EN_WORDLIST } from "./bip39EnWordlist"
 

--- a/packages/hdkd-helpers/src/utils.ts
+++ b/packages/hdkd-helpers/src/utils.ts
@@ -1,4 +1,4 @@
-import { blake2b } from "@noble/hashes/blake2"
+import { blake2b } from "@noble/hashes/blake2.js"
 import { hexToBytes, isBytes } from "@noble/hashes/utils.js"
 import { Bytes, str, Tuple } from "scale-ts"
 import type { DeriveKeyPairFn } from "./internal/types"

--- a/packages/hdkd-helpers/test/hard-derivation-to-public-key.test.ts
+++ b/packages/hdkd-helpers/test/hard-derivation-to-public-key.test.ts
@@ -1,4 +1,4 @@
-import { bytesToHex } from "@noble/hashes/utils"
+import { bytesToHex } from "@noble/hashes/utils.js"
 import {
   sr25519_derive_seed_hard,
   sr25519_secret_from_seed,

--- a/packages/hdkd-helpers/test/mini-secret-to-public-key.test.ts
+++ b/packages/hdkd-helpers/test/mini-secret-to-public-key.test.ts
@@ -1,4 +1,4 @@
-import { bytesToHex } from "@noble/hashes/utils"
+import { bytesToHex } from "@noble/hashes/utils.js"
 import { sr25519_secret_from_seed } from "@polkadot-labs/schnorrkel-wasm"
 import { expect, test } from "vitest"
 import { DEV_MINI_SECRET } from "../src/constants"

--- a/packages/hdkd-helpers/test/mixed-derivation-to-public-key.test.ts
+++ b/packages/hdkd-helpers/test/mixed-derivation-to-public-key.test.ts
@@ -1,4 +1,4 @@
-import { bytesToHex } from "@noble/hashes/utils"
+import { bytesToHex } from "@noble/hashes/utils.js"
 import {
   sr25519_derive_keypair_hard,
   sr25519_derive_keypair_soft,

--- a/packages/hdkd-helpers/test/soft-derivation-to-public-key.test.ts
+++ b/packages/hdkd-helpers/test/soft-derivation-to-public-key.test.ts
@@ -1,4 +1,4 @@
-import { bytesToHex } from "@noble/hashes/utils"
+import { bytesToHex } from "@noble/hashes/utils.js"
 import {
   sr25519_derive_public_soft,
   sr25519_secret_from_seed,

